c_src/sqlite3_nif.c

```diff
@@ -810,6 +898,13 @@ on_load(ErlNifEnv* env, void** priv, ERL_NIF_TERM info)
     return 0;
 }
 
+static void
+on_unload(ErlNifEnv* caller_env, void* priv_data)
```

Comment:
```suggestion
on_unload(ErlNifEnv* env, void* priv_data)
```

Reply:
We are going to leave this as `caller_env` as that is what it is defined as within the erl_nif documentation. https://www.erlang.org/doc/man/erl_nif.html#initialization